examples/backtest.ipynb

```diff
@@ -23,15 +23,33 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    from etna.models import CatBoostModelMultiSegment
</pre><p>In fact you can use prophet here to simplify the code, as it doesn't produce tones of logs now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>You imported the prophet, but still using catboost)))</p>

Reply:
<p>May you use prophet here without any features</p>

Reply:
<p>Remove the redundant imports as you don't use trnsforms</p>